### PR TITLE
Upgrade to generator-jhipster master before v6.9.0

### PR DESCRIPTION
**Pull request by pascalgrimaud**
-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/jhipster-vuejs/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/entity-client/templates/vue/src/main/webapp/app/entities/entity-update.vue.ejs [hunk 1]
```diff
@@ -37,10 +37,10 @@
...
                         <select class="form-control" name="<%= fieldName %>" :class="{'valid': !$v.<%= entityInstance %>.<%= fieldName %>.$invalid, 'invalid': $v.<%= entityInstance %>.<%= fieldName %>.$invalid }" v-model="$v.<%= entityInstance %>.<%= fieldName %>.$model" id="<%= entityFileName %>-<%= fieldName %>" <% if (fields[idx].fieldValidate === true && fields[idx].fieldValidateRules.includes('required')) { %> required<% } %>>
                         <%_ const enumPrefix = angularAppName + '.'+ fieldType;
-                            const values = fields[idx].fieldValues.replace(/\s/g, '').split(',');
+                            const values = getEnumValuesWithCustomValues(fields[idx].fieldValues);
                             for (key in values) {
-                                const value = values[key]; _%>
-                            <option value="<%= value %>" v-bind:label="$t('<%=enumPrefix%>.<%=value%>')"><%=value%></option>
+                                const enumValue = values[key]; _%>
+                            <option value="<%= enumValue.name %>" v-bind:label="$t('<%=enumPrefix%>.<%= enumValue.name %>')"><%= enumValue.name %></option>
```
Comment (mshima) [inline generators/entity-client/templates/vue/src/main/webapp/app/entities/entity-update.vue.ejs:43]:
```suggestion
                            <option value="<%= enumValue.name %>" v-bind:label="$t('<%=enumPrefix%>.<%= enumValue.value %>')"><%= enumValue.value %></option>
```
Correct?

Comment (pascalgrimaud) [reply generators/entity-client/templates/vue/src/main/webapp/app/entities/entity-update.vue.ejs]: Thanks @mshima : I'm trying this and commiting your suggestion

**Comment by pascalgrimaud** [comment]
Thanks @atomfrede 
So as soon as this PR is merged and your other PR is merged, we'll be ready to release Vue.js blueprint, just after the release of main generator-jhipster v6.9.0 :)

**Comment by atomfrede** [comment]
Thats great!

**Pull request merged**